Sources/SwifterSwift/UIKit/UIApplicationExtensions.swift

```diff
@@ -55,17 +55,17 @@ public extension UIApplication {
 
     /// SwifterSwift: Application name (if applicable).
     var displayName: String? {
-        return Bundle.main.infoDictionary?[kCFBundleNameKey as String] as? String
+        return Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as? String
```

Comment:
Is there a constant for this raw string "CFBundleDisplayName"?

Reply:
I wish there were but no.